index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <script src="http://cdn.mathjax.org/mathjax/2.3-latest/MathJax.js?config=MML_HTMLorMML"></script>
```

Comment:
MathJax claims on their site that you can install it with Bower (and I assume that means you can serve it with gulp?) since version 2.3, so you could give that a try.